src/commands/CmdEdit.cpp

```diff
@@ -667,7 +667,8 @@ void CmdEdit::parseTask (Task& task, const std::string& after, const std::string
   value = findValue (after, "\n  Dependencies:");
   auto dependencies = split (value, ',');
 
-  task.remove ("depends");
+  for (auto& dep : task.getDependencyUUIDs ())
+    task.removeDependency (dep);
```

Comment:
This works fine for now, but I wonder if we want to put more sophisticated logic in place. Removing all dependencies and then re-adding some of them will potentially generate a lot of operations that cancel each other out (removal and addition of the same `dep_X` attribute).

Reply:
Yeah, I think the "edit" functionality will need a good going-over when that happens.